tests/unit/helpers/dates/companiDuration.test.js

```diff
@@ -39,6 +39,32 @@ describe('CompaniDuration', () => {
       sinon.assert.calledWithExactly(_formatMiscToCompaniDuration.getCall(0), duration);
     });
 
+    it('should return duration from iso', () => {
+      const result = CompaniDurationsHelper.CompaniDuration('P1Y2M2DT3H4M2S');
+
+      expect(result)
+        .toEqual(expect.objectContaining({
+          _getDuration: expect.any(luxon.Duration),
+          format: expect.any(Function),
+          add: expect.any(Function),
+          asHours: expect.any(Function),
+        }));
+      sinon.assert.calledWithExactly(_formatMiscToCompaniDuration.getCall(0), 'P1Y2M2DT3H4M2S');
```

Comment:
tu peux t'assurer que la valeur est bonne, comme pour le test 'should return duration if arg is an object with luxon duration key'

genre `expect(new luxon.Duration(result).toMillis()).toEqual(3 * 60 * 60 * 1000 + 35 * 60 * 1000 + 12 * 1000);`